frontend-v2/src/features/data/LoadDataStepper.tsx

```diff
@@ -261,7 +267,7 @@ const LoadDataStepper: FC<IStepper> = ({ csv = "", onCancel, onFinish }) => {
         width: "100%",
       }}
     >
-      <Stepper nonLinear activeStep={stepState.activeStep} alternativeLabel>
+      <Stepper nonLinear activeStep={stepState.activeStep}>
```

Comment:
I think I used`alternativeLabel` here to make the step labels clickable, but I don't know if that's still needed in MUI v6.

Reply:
Looks like the step button still work, without `alternativeLabel`.

Reply:
My mistake: I used `nonLinear` to make the steps clickable. Ignore my previous comments.